packages/ui/components/Drawer.tsx

```diff
@@ -18,6 +18,7 @@ interface DrawerProps {
   expanded?: boolean;
   handleCloseClick?: () => void;
   closeDrawer: () => void;
+  handleLogout?: () => void;
```

Comment:
You don't need to pass this as a prop to Drawer. You should just import this function from your auth module and pass it into the Logout specific `NavLink` as a prop below.